transport-native-epoll/src/main/java/io/netty/channel/epoll/AbstractEpollChannel.java

```diff
@@ -33,6 +33,7 @@
 abstract class AbstractEpollChannel extends AbstractChannel {
     private static final ChannelMetadata DATA = new ChannelMetadata(false);
     private final int readFlag;
+    private final FileDescriptor fileDescriptor;
     protected int flags;
     protected volatile boolean active;
     volatile int fd;
```

Comment:
Is there a reason we have two fields for file descriptor? (`fileDescriptor` and `fd`) I feel like we can do the same with just a single field.